xbmc/utils/StringUtils.cpp

```diff
@@ -591,6 +591,39 @@ std::string& StringUtils::RemoveDuplicatedSpacesAndTabs(std::string& str)
   return str;
 }
 
+bool StringUtils::IsSpecialCharacter(char c)
+{
+  constexpr std::string_view strVew(" .-_+,!'\"\t/\\*?#$%&@()[]{}");
```

Comment:
`static constexpr` please. Otherwise the instance of `strVew` will be created whenever you enter the method.